colabs/raytune/W&B + RayTune.ipynb

```diff
@@ -6,7 +6,8 @@
       "name": "W&B-RayTune.ipynb",
```

Comment:
<p>I would make this less prominent. Maybe a level 2 or level 3 heading (<code>##</code>  or <code>###</code> )</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>